src/components/Tabs/Tabs.scss

```diff
@@ -236,9 +236,13 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
   border-radius: var(--p-border-radius-base);
   &:hover {
     background-color: var(--p-surface-hovered);
+
+    @media (-ms-high-contrast: active) {
```

Comment:
We might be able to leverage the `high-contrast-outline` mixin here and a few other spots. 

https://github.com/Shopify/polaris-react/blob/49a6d0de7057b3579ce9a59608a6eb942a86afe1/src/styles/foundation/_accessibility.scss